README.md

```diff
@@ -1,2 +1,46 @@
-# codeowners-generator-action
-automatically fix your codeowners file when using codeowners generator
+<p align="center">
```

Comment:
we need to change copy here and add some clear directions and how to use the workflow